package.json

```diff
@@ -13,6 +13,7 @@
 		"eslint": "eslint . --ext js --ext jsx --ext ts --ext tsx"
 	},
 	"dependencies": {
+		"add-to-calendar-button": "^1.9.0",
```

Comment:
We can remove this you can run `num uninstall add-to-calendar-button` or whatever the package name was